src/components/Crossroad/Crossroad.tsx

```diff
@@ -0,0 +1,54 @@
+import {Box, Stack, Typography} from '@mui/material'
+import Grid from '@mui/material/Unstable_Grid2'
+import React, {useState} from 'react'
+
+import Strom from '@/svg/strom.svg'
+
+import {Button} from '../Clickable/Button'
+import {Link} from '../Clickable/Link'
+
+export const Crossroad: React.FC = () => {
+  const [isProblem, setIsProblem] = useState(false)
+
+  const corssroadItems = [
+    {label: 'STROM', url: 'strom/'},
+    {label: 'Matik', url: 'matik/'},
+    {label: 'Malynár', url: 'malynar/'},
+    {label: 'Kôš', url: 'https://kos.strom.sk/'},
+    {label: 'Máš problém', url: 'https://masproblem.strom.sk/'},
+  ]
+  return !isProblem ? (
+    <Stack justifyContent="center" alignItems="center" height="100vh" sx={{bgcolor: 'black'}}>
```

Comment:
v PageLayout mame 100dvh kvoli mobilu, treba skusit na device, ci to nerobi zbytocne scrollbary... ale dvh bude asi safer